recipes/imas-data-dictionary/recipe.yaml

```
@@ -0,0 +1,62 @@
+context:
+  version: "4.0.1"
+  name: "imas-data-dictionary"
+  build: 0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name | split("-") | join("_") }}-${{ version }}.tar.gz
+  sha256: 89327fabc71cd06772833c04dab06800ed3ac41ec7b9a671227881d429690fe7
+
+build:
+  number: ${{ build }}
+  noarch: python
+  python:
+    entry_points:
+      - idsinfo = data_dictionary.idsinfo:main
+  files:
+    include: ["**"]
+    exclude:
+      - dd_${{ version }}/share/
+
+requirements:
+  build:
+    - saxon-he
+    - libxslt
+  host:
+    - pip
+    - python ${{ python_min }}.*
+    - setuptools >=64
+    - setuptools-scm
+  run:
+    - python >=${{ python_min }}
+    - packaging
+  run_exports:
+    - ${{ pin_subpackage("imas-data-dictionary", lower_bound="x.x", upper_bound="x.x") }}
+
+tests:
+  - python:
+      imports:
+        - data_dictionary
+      pip_check: false
```

```suggestion
      pip_check: false
      requirements:
        run:
          - python ${{ python_min }}
```
Can you explain why you are skipping `pip check` here?  Also you need to add a run requirement here for the linter to pass.

---

The explanation of why you skip `pip_check` here is still missing.